playbooks/openstack-osh_hostconfig.yml

```diff
@@ -8,6 +8,25 @@
     - name: Ensure all the openstack mechanism vars are loaded
       include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
 
+    - name: Collect repo files for removal
+      find:
+        paths: "/etc/zypp/repos.d/"
+        patterns: "*.repo"
+      register: repo_files
+
+    - name: Clear existing repo config
```

Comment:
Could we disable them instead? Removing them may bring issues with existing repos on the deployer.

Or maybe move them somewhere?

Reply:
There are no other existing repos on the deployer than what's part of the image. And those are the ones we want to replace. What other repos are there that we'd want to keep?

Reply:
Im just thinking of someone bringing its own deployer, instead of letting us create it, but that is not supported, nor this is logical as there may as well be other changes to do in that case, so ignore me!